### PR TITLE
[Improve][CI]skip ui module, improve module dependent

### PR DESCRIPTION
**Pull request by hawk9821**
### Purpose of this pull request
ci subtasks save time by skipping front-end packaging 
c97bf8f0032732cda383796c8c7a593

Improve module dependent  before

```
2024-12-13T23:46:45.1019896Z [INFO] SeaTunnel : Engine : Server ........................ SUCCESS [  5.357 s]
2024-12-13T23:46:45.1020284Z [INFO] SeaTunnel : Engine : Client ........................ SUCCESS [  1.701 s]
2024-12-13T23:46:45.1020678Z [INFO] SeaTunnel : Core : Starter ......................... SUCCESS [  6.983 s]
2024-12-13T23:46:45.1021083Z [INFO] SeaTunnel : Formats : Compatible Debezium Json ..... SUCCESS [  1.053 s]
2024-12-13T23:46:45.1021491Z [INFO] SeaTunnel : Connectors V2 : CDC : .................. SUCCESS [  0.078 s]
2024-12-13T23:46:45.1021895Z [INFO] SeaTunnel : Connectors V2 : CDC : Base ............. SUCCESS [  2.200 s]
2024-12-13T23:46:45.1022292Z [INFO] SeaTunnel : Connectors V2 : Jdbc ................... SUCCESS [ 16.994 s]
2024-12-13T23:46:45.1022697Z [INFO] SeaTunnel : Connectors V2 : CDC : MySql ............ SUCCESS [ 13.442 s]
2024-12-13T23:46:45.1023193Z [INFO] SeaTunnel : Connectors V2 : Assert ................. SUCCESS [  1.506 s]
2024-12-13T23:46:45.1023882Z [INFO] SeaTunnel : Connectors V2 : Paimon ................. SUCCESS [  9.935 s]
2024-12-13T23:46:45.1024290Z [INFO] SeaTunnel : E2E : Connector V2 : ................... SUCCESS [  0.055 s]
2024-12-13T23:46:45.1024675Z [INFO] SeaTunnel : E2E : Engine : ......................... SUCCESS [  0.170 s]
2024-12-13T23:46:45.1025148Z [INFO] SeaTunnel : E2E : Engine : Base .................... SUCCESS [41:12 min]
2024-12-13T23:46:45.1025544Z [INFO] SeaTunnel : E2E : Connector V2 : Paimon ............ SUCCESS [32:18 min]
2024-12-13T23:46:45.1025919Z [INFO] ------------------------------------------------------------------------
2024-12-13T23:46:45.1026207Z [INFO] BUILD SUCCESS
2024-12-13T23:46:45.1026453Z [INFO] ------------------------------------------------------------------------
2024-12-13T23:46:45.1026740Z [INFO] Total time:  01:18 h
2024-12-13T23:46:45.1026971Z [INFO] Finished at: 2024-12-13T23:46:45Z
2024-12-13T23:46:45.1027266Z [INFO] ------------------------------------------------------------------------
```
after

```
2024-12-14T09:52:07.4090505Z [INFO] SeaTunnel : Engine : Client ........................ SUCCESS [  1.689 s]
2024-12-14T09:52:07.4091483Z [INFO] SeaTunnel : Core : Starter ......................... SUCCESS [  6.913 s]
2024-12-14T09:52:07.4092298Z [INFO] SeaTunnel : Formats : Compatible Debezium Json ..... SUCCESS [  1.077 s]
2024-12-14T09:52:07.4093277Z [INFO] SeaTunnel : Connectors V2 : CDC : .................. SUCCESS [  0.079 s]
2024-12-14T09:52:07.4094098Z [INFO] SeaTunnel : Connectors V2 : CDC : Base ............. SUCCESS [  2.743 s]
2024-12-14T09:52:07.4094938Z [INFO] SeaTunnel : Connectors V2 : Jdbc ................... SUCCESS [ 16.688 s]
2024-12-14T09:52:07.4095763Z [INFO] SeaTunnel : Connectors V2 : CDC : MySql ............ SUCCESS [ 13.668 s]
2024-12-14T09:52:07.4096584Z [INFO] SeaTunnel : Connectors V2 : Assert ................. SUCCESS [  1.483 s]
2024-12-14T09:52:07.4097576Z [INFO] SeaTunnel : Connectors V2 : Paimon ................. SUCCESS [  8.699 s]
2024-12-14T09:52:07.4098384Z [INFO] SeaTunnel : E2E : Connector V2 : ................... SUCCESS [  0.057 s]
2024-12-14T09:52:07.4099194Z [INFO] SeaTunnel : E2E : Connector V2 : Paimon ............ SUCCESS [33:15 min]
2024-12-14T09:52:07.4099953Z [INFO] ------------------------------------------------------------------------
2024-12-14T09:52:07.4100536Z [INFO] BUILD SUCCESS
2024-12-14T09:52:07.4101029Z [INFO] ------------------------------------------------------------------------
2024-12-14T09:52:07.4101928Z [INFO] Total time:  37:31 min
2024-12-14T09:52:07.4102395Z [INFO] Finished at: 2024-12-14T09:52:07Z
2024-12-14T09:52:07.4102970Z [INFO] ------------------------------------------------------------------------
```

### Check list

* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update [plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  2. Update the pom file of [seatunnel-dist](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)
  3. Add ci label in [label-scope-conf](https://github.com/apache/seatunnel/blob/dev/.github/workflows/labeler/label-scope-conf.yml)
  4. Add e2e testcase in [seatunnel-e2e](https://github.com/apache/seatunnel/tree/dev/seatunnel-e2e/seatunnel-connector-v2-e2e/)
  5. Update connector [plugin_config](https://github.com/apache/seatunnel/blob/dev/config/plugin_config)
* [ ] Update the [`release-note`](https://github.com/apache/seatunnel/blob/dev/release-note.md).

### CODE REVIEW

**Comment by Hisoka-X** [comment]
Please  rebase from dev.

**Review by Hisoka-X — DISMISSED** [review]
Thanks @hawk9821

File: seatunnel-e2e/seatunnel-connector-v2-e2e/connector-cdc-oracle-e2e/src/test/java/org/apache/seatunnel/connectors/seatunnel/cdc/oracle/AbstractOracleCDCIT.java [hunk 1]
```diff
@@ -214,6 +214,9 @@ protected static void dropTable(String jdbcUrl, String schemaName, String tableN
                                     schemaName, tableName));
             if (resultSet.next()) {
                 statement.execute(String.format("DROP TABLE %s.%s", schemaName, tableName));
+                connection
+                        .createStatement()
+                        .execute(String.format("DROP TABLE %s.%s", schemaName, tableName));
```
Comment (hailin0) [inline seatunnel-e2e/seatunnel-connector-v2-e2e/connector-cdc-oracle-e2e/src/test/java/org/apache/seatunnel/connectors/seatunnel/cdc/oracle/AbstractOracleCDCIT.java]: Duplicate execute？

Comment (hawk9821) [reply seatunnel-e2e/seatunnel-connector-v2-e2e/connector-cdc-oracle-e2e/src/test/java/org/apache/seatunnel/connectors/seatunnel/cdc/oracle/AbstractOracleCDCIT.java]: rebased

**Review by Hisoka-X — APPROVED** [review]

**Review by hailin0 — APPROVED** [review]

**Pull request merged**